mantidproject/mantid
# attempt at fixing muon autoscale bug

**Description of work.**


It was possible to create a strange visual bug in the muon GUI as the fitting plotting pane was not autoscaling correctly when the data was changed.
**To test:**
Note: you may have to turn the autoscale off and on again (this is a glitch from doing strange plots). 

Load MUSR 62260
Set the plot to tiled
Go to the corrections tab
Turn on the background corrections and set them to manual 
Untick the apply to all option
Change one of the background values to 10000
You will see one of the data sets now shoots up
Go to the fitting tab
Turn the tiled plot on
Load the next run by pressing the `>` button at the top of the GUI (to load 62261)
Check that one plot goes to large y values and the other 3 are at about -0.4 to 0.4


*There is no associated issue.*






---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
